spec/default_facts.yml

```diff
@@ -2,7 +2,8 @@
 #
 # Facts specified here will override the values provided by rspec-puppet-facts.
 ---
-ipaddress: "172.16.254.254"
-ipaddress6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
+networking:
```

Comment:
this breaks a lot of stuff, maybe also the tests in this module (see https://github.com/puppetlabs/pdk-templates/issues/543).  If that's the case we should probably fix it manually.

Reply:
file is not in the PR.